docs/content/user-guides/crd-acme/03-deployments.yml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: traefik-ingress-controller
+
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: traefik
+  labels:
+    app: traefik
+
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: traefik
+  template:
+    metadata:
+      labels:
+        app: traefik
+    spec:
+      serviceAccountName: traefik-ingress-controller
```

Comment:
Similarly, since the SA is namespaced, we have to match, and add the namespace to this object too

Reply:
ok, I think I've addressed your comment while not breaking anything. PTAL.

Reply:
Usually I would put the namespace on the deployment object, not on the pod spec. If the deployment is created in another namespace, it could get weird.

Reply:
I figured it out. The error I was getting, when adding a namespace on the deployment object i.e.: "failed to process /var/lib/rancher/k3s/server/manifests/03-deployments.yml: got extensions/v1beta1, Kind=Deployment objects that are both namespaced and not namespaced"
was because then we have to add a namespace on the _other_ deployment object (the whoami one) as well.